src/application/controller/controller.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONTROLLER_H_
+#define CONTROLLER_H_
+
+#include "FreeRTOS.h"
+#include "state_estimation.h"
+#include <stdbool.h>
+
+/* Enums/Types */
+typedef struct {
+    float x;
+    float y;
+    float z;
+} vector3_t;
+
+// main controller state
+typedef struct {
+    vector3_t attitude; // Current attitude vector
+    vector3_t rates; // Current angular rates
+    vector3_t velocity; // Current velocity vector
+    float altitude; // Current altitude
+    float timestamp; // Timestamp in ms
+} controller_state_t;
+
+// Output of controller to be sent to servo via CAN
+typedef struct {
+    float commanded_angle; // Desired angle
+    float actual_angle; // Current angle
+    uint32_t timestamp; // Timestamp
+    uint32_t servo_id; // Servo identifier
```

Comment:
Can you explain what servo_id is for? We only have one servo. @rexblade21 I'd like you to work with processor ppl to figure out what u want in your message